CMakeLists.txt

```diff
@@ -471,11 +471,11 @@ if(CMAKE_CXX_COMPILER_ID MATCHES Clang)
 		-Weverything
 		-Wno-documentation
 		-Wno-c++98-compat-pedantic
+		-Wno-c++11-compat-pedantic
 		-Wno-padded
 		-Wno-global-constructors
 		-Wno-exit-time-destructors
 		-Wno-weak-vtables
-		-Wno-return-std-move-in-c++11
```

Comment:
it seem you need to leave this one here a well: https://travis-ci.org/github/arvidn/libtorrent/jobs/701265066#L1407

Reply:
fixed